demo/index.html

```diff
@@ -30,17 +30,17 @@
                     "marginCountryPopupForLeft" : 60
                 },
                 'colors': {
-                    "hover" : "red",
-                    "highlighted": "yellow"
+                    "hovercoloroption" : "orange",
+                    "highlightedcoloroption": "yellow"
                 }
             };
-            $('#world-map').interactiveWorldMap()
+            $('#world-map').interactiveWorldMap(options)
         });
     </script>
 
 </head>
 <body>
-<div id="world-map" data-colorHighlighted="red" data-colorHover="blue" data-countries='{ "countries": {
+<div id="world-map" data-highlightedcoloroption="red" data-hoverColorOption="blue" data-countries='{ "countries": {
```

Comment:
Also instead of setting ervy single option with a convoluted name, use hahes!
```
data-colors=' { "hover" : "red", "highlighted" : "blue" }
```

Much clearer, and also re-usable (see comment above)